### PR TITLE
Added Document for VMware Tanzu Application Service

### PR DESCRIPTION
**Pull request by akhil-sumologic**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request...

Jira project
https://sumologic.atlassian.net/browse/SUMO-237234
## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

**Comment by akhil-sumologic** [comment]
@JV0812 , As discussed, there will be minor changes.

**Comment by kimsauce** [comment]
Hi @akhil-sumologic , thank you for your contribution! As this is an open source project, we require contributors to sign our Contributor License Agreement and do not have yours on file. To proceed with your PR, please [sign your name here](https://forms.gle/YgLddrckeJaCdZYA6) and we'll add you to our approved list of contributors.

File: docs/integrations/containers-orchestration/vmware-tanzu-application-service.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+sidebar_label: VMware Tanzu Application Service
+description: This documentation describes the Sumo Logic Nozzle for VMware Tanzu tile.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/containers-orchestration/vmware-tanzu.png')} alt="VMware Tanzu Application Service" width="50" />
+
+## <a id='nozzle-overview'></a> Overview of Sumo Logic Nozzle for VMware Tanzu
+
+Sumo Logic Nozzle for VMware Tanzu can help you:
```
Comment (kimsauce) [inline docs/integrations/containers-orchestration/vmware-tanzu-application-service.md:13]: @akhil-sumologic , our docs style is to omit the "overview" header and go straight into the overview description

```suggestion
The Sumo Logic Nozzle for VMware Tanzu can help you:
```

File: docs/integrations/containers-orchestration/vmware-tanzu-application-service.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+<img src={useBaseUrl('img/integrations/containers-orchestration/vmware-tanzu.png')} alt="VMware Tanzu Application Service" width="50" />
+
+The Sumo Logic Nozzle for VMware Tanzu can help you:
+
+* View dashboards to monitor the overall health and security of your VMware Tanzu resources
+* Drill down from alerts to see key performance metrics and audit logs for VMware Tanzu and your custom apps
+* Store VMware Tanzu audit details in a fully secure SOC 2, HIPAA, and PCI-compliant environment
+
+The Sumo Logic nozzle for [VMware Tanzu Firehose](https://docs.vmware.com/en/Tile-Developer-Guide/3.0/tile-dev-guide/nozzle.html) allows operations teams and app owners to monitor and troubleshoot the VMware Tanzu platform and apps in real time, without having to install a Sumo Logic agent in their Cloud Foundry environment.
```
Comment (kimsauce) [inline docs/integrations/containers-orchestration/vmware-tanzu-application-service.md:18]:
```suggestion
The Sumo Logic Nozzle for [VMware Tanzu Firehose](https://docs.vmware.com/en/Tile-Developer-Guide/3.0/tile-dev-guide/nozzle.html) allows operations teams and app owners to monitor and troubleshoot the VMware Tanzu platform and apps in real time, without having to install a Sumo Logic agent in their Cloud Foundry environment.
```

**Comment by akhil-sumologic** [comment]
> Hi @akhil-sumologic , thank you for your contribution! As this is an open source project, we require contributors to sign our Contributor License Agreement and do not have yours on file. To proceed with your PR, please [sign your name here](https://forms.gle/YgLddrckeJaCdZYA6) and we'll add you to our approved list of contributors.

File: docs/integrations/containers-orchestration/vmware-tanzu-application-service.md (hunk 1 above)
Comment (akhil-sumologic) [reply docs/integrations/containers-orchestration/vmware-tanzu-application-service.md]: > @akhil-sumologic , our docs style is to omit the "overview" header and go straight into the overview description

Addressed

**Comment by akhil-sumologic** [comment]
> Hi @akhil-sumologic , thank you for your contribution! As this is an open source project, we require contributors to sign our Contributor License Agreement and do not have yours on file. To proceed with your PR, please [sign your name here](https://forms.gle/YgLddrckeJaCdZYA6) and we'll add you to our approved list of contributors.

i have added myself.

**Comment by jpipkin1** [comment]
@akhil-sumologic, our **verification/cla-signed** check is failing. Did you follow the instructions in the above comment box?

> Thank you for your pull request and welcome to our community. We could not parse the GitHub identity of the following contributors: **Akhil Dangore**.
> This is most likely caused by a git client misconfiguration; please make sure to:
> 1. check if your git client is configured with an email to sign commits git config --list | grep email
> 2. If not, set it up using git config --global user.email email@example.com
> 3. Make sure that the git commit email is configured in your GitHub account settings, see https://github.com/settings/emails

**Comment by akhil-sumologic** [comment]
> @akhil-sumologic, our **verification/cla-signed** check is failing.
> 
> Did you follow the instructions in the above comment box?
> 
> > Thank you for your pull request and welcome to our community. We could not parse the GitHub identity of the following contributors: **Akhil Dangore**.
> > This is most likely caused by a git client misconfiguration; please make sure to:
> > 
> > 1. check if your git client is configured with an email to sign commits git config --list | grep email
> > 2. If not, set it up using git config --global user.email [email@example.com](mailto:email@example.com)
> > 3. Make sure that the git commit email is configured in your GitHub account settings, see https://github.com/settings/emails

Yes, configured email

Screenshot 2024-03-06 at 8 40 12 PM

Screenshot 2024-03-06 at 8 43 03 PM

**Comment by jpipkin1** [comment]
Thank you, @akhil-sumologic.

@kimsauce, do you have any idea why the email parsing is not working?

**Comment by jpipkin1** [comment]
@akhil-sumologic, I apologize for the problem with GitHub identity parsing. 

To work around the problem, I'm closing this PR and replacing it with PR #3786.

**Pull request closed without merging**